static/app/components/badge.tsx

```diff
@@ -27,7 +27,6 @@ const Badge = styled(({children, text, ...props}: Props) => (
   transition: background 100ms linear;
 
   position: relative;
-  top: -1px;
```

Comment:
This was added to better align `<Badge />` inside nav tabs.

With `top: -1px`:
<img width="352" alt="Screen Shot 2022-03-28 at 1 31 15 PM" src="https://user-images.githubusercontent.com/44172267/160481938-a98f4373-f8ae-4c21-8940-8655d0eee989.png">

Without:
<img width="352" alt="Screen Shot 2022-03-28 at 1 31 30 PM" src="https://user-images.githubusercontent.com/44172267/160481974-f6d038ef-4228-401a-967e-aa6dc7773b34.png">

However, this results in `<Badge />` being misaligned everywhere else. Better to remove this and center-align the content inside nav tabs (https://github.com/getsentry/sentry/pull/33002/commits/aeb6b7aa4284cc1d061c7f28eb48b8d3c9fb2482).

Reply:
👍  nice